test/clj/cider/nrepl/middleware/inspect_test.clj

```diff
@@ -505,3 +515,17 @@
                                               :var-name "sub-map"})
                             (session/message {:op   "eval"
                                               :code "sub-map"}))))))))
+
+(deftest inspect-tap-current-value-test
+  (testing "inspect-tap-current-value taps the current inspector value"
+    (do
+      (session/message {:op   "eval"
+                        :code "(def x (+ 3 4)))"})
+      (session/message {:op "eval"
+                        :inspect "true"
+                        :code    "x"})
+      (session/message {:op  "inspect-push"
+                        :idx 1})
+      (session/message {:op  "inspect-tap-current-value"}))
+    (Thread/sleep 10000)
```

Comment:
Please sleep in chunks of 50ms, up 1o 10000

Reply:
Not sure what you mean. Just wait using a loop ?

Or testing in a loop, with 50 ms waiting in between and fail if limit reached ?